src/store/ProjectsStore.js

```diff
@@ -3,13 +3,13 @@ import apiClient from 'panoptes-client/lib/api-client.js'
 import ASYNC_STATES from 'helpers/asyncStates'
 
 const ROLES = {
-  RESEARCHER: 'Researcher',
-  VOLUNTEER: 'Volunteer',
+  ADMIN: 'Admin',
+  EDITOR: 'Editor',
   VIEWER: 'Viewer'
 }
 
-const researcherRoles = ['owner', 'collaborator']
-const volunteerRoles = ['expert', 'researcher', 'moderator']
+const adminRoles = ['owner', 'collaborator']
+const editorRoles = ['expert', 'researcher', 'moderator']
```

Comment:
Oh I think I see why it's still happening! I think `'scientist'` needs to be added to this array, per 
![Screen Shot 2020-02-26 at 10 35 16 AM](https://user-images.githubusercontent.com/12118751/75366033-bcd11300-5883-11ea-929d-69574f161d3c.png)
Oy the roles are a quagmire! But I think if this should do it.

Reply:
Panoptes defines the "researcher" role as a "scientist?" That's very confusing. I'll make the edit and ask you to give it one more spin to make sure all is sorted out. Thanks for your tedious review.